velox/dwio/parquet/tests/reader/ParquetReaderTest.cpp

```diff
@@ -630,3 +630,44 @@ TEST_F(ParquetReaderTest, preloadSmallFile) {
     ASSERT_EQ(file->bytesRead(), 0);
   }
 }
+
+TEST_F(ParquetReaderTest, prefetchRowGroups) {
+  auto rowType = ROW({"id"}, {BIGINT()});
+  const std::string sample(getExampleFilePath("multiple_row_groups.parquet"));
+  const int numRowGroups = 4;
+
+  ReaderOptions readerOptions{defaultPool.get()};
+  // Disable preload of file.
+  readerOptions.setFilePreloadThreshold(0);
+  readerOptions.setPrefetchRowGroups(1);
```

Comment:
Should we test some other values as well? E.g. 2 (less than total), 4 (exactly as total), 10 (more than total)?